src/splitInteger.test.js

```diff
@@ -2,20 +2,44 @@
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
+test('should return equal parts', () => {
```

Comment:
The test descriptions should be more meaningful and specific to help understand what's being tested in a better way.

Reply:
Done